README.md

```diff
@@ -16,16 +12,26 @@ not undergone thorough testing/use yet.
 
  Name       | Setoid  | Semigroup | Functor | Applicative | Monad | Comonad  | Foldable |
```

Comment:
What about linking these column headers to their definitions in the Fantasy Land spec eg [Setoid](https://github.com/fantasyland/fantasy-land#setoid)